int32 modeId
geometry_msgs/Point waypointPosition
geometry_msgs/Pose robotPose
float32 heading
float32 speed
---
int32 actionId
